css/style.css

```diff
@@ -1,11 +1,362 @@
   background-color: var(--primary-color);
   transform: scale(0.9);
   object-fit: contain;
 }
+
 .section-category .img {
+  transition: all 0.1s linear;
   background-color: var(--dark-color-7);
   object-fit: contain;
 }
+.hero-section-button {
+  padding: 0.8rem 1rem;
+}
+
+.hero-image {
+  display: flex;
+  justify-content: center;
+}
+.hero-section {
+  display: flex;
+  width: 100%;
+  padding: 4rem 6rem;
+  align-items: center;
+  justify-content: space-between;
+}
+@media screen and (max-width: 1200px) {
+  .hero-section-content {
+    order: 2;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    align-items: center;
+  }
+  .hero-image {
+    order: 1;
+  }
+  .hero-section {
+    width: 100%;
+    gap: 3rem;
+    flex-direction: column;
+  }
+  .section-category,
+  .section-brands {
+    text-align: center;
+    justify-content: center;
+  }
+  .section-category .row {
+    justify-content: center;
+  }
+  .section-brands .row {
+    justify-content: center;
+  }
+}
+
+li a.btn {
+  font-size: 0.9rem;
+  padding: 0.4rem 0.6rem;
+}
+.browse-more-btn {
+  padding: 0.4rem 0.6rem;
+  font-size: 0.75rem;
+  line-height: 1.25rem;
+}
+form.form {
+  background-color: var(--light-grey);
+  margin-bottom: 15vh;
+  margin-top: 0vh;
+}
+@media (max-width: 535px) {
+  form.form {
+    padding: 1rem;
+    margin: 1rem;
+  }
+}
+input[type="search"]::-webkit-search-cancel-button {
+  -webkit-appearance: none;
+  height: 1em;
+  width: 1em;
+  border-radius: 50em;
+  background: url(https://pro.fontawesome.com/releases/v5.10.0/svgs/solid/times-circle.svg)
+    no-repeat 50% 50%;
+  background-size: contain;
+  opacity: 0;
+  pointer-events: none;
+  filter: invert(1);
+}
+
+input[type="search"]:focus::-webkit-search-cancel-button {
+  opacity: 0.3;
+  pointer-events: all;
+}
+
+main {
+  height: 100vh;
+}
+.sidebar {
+  margin-top: 2vh;
+  padding-bottom: 12vh;
+  flex: 3;
+  margin: 2rem;
+  margin-top: 0;
+  height: 100vh;
+  flex-wrap: nowrap;
+  overflow: hidden;
+  overflow-y: scroll;
+  padding-bottom: 25vh;
+
+  border-right: 2px solid var(--primary-color);
+}
+.sidebar .col {
+  padding: 0.5rem;
+}
+section.products {
+  flex: 12;
+  height: 100vh;
+  overflow: hidden;
+  overflow-y: scroll;
+  padding-left: 2%;
+  padding-bottom: 25vh;
+}
+.filter-btn {
+  display: none;
+}
+@media (max-width: 950px) {
+  section.products .row:nth-child(2) {
+    justify-content: center;
+
+    width: 100vw;
+  }
+  section.products .row:nth-child(1) {
+    padding: 0 2rem;
+
+    width: 100vw;
+  }
+  .sidebar {
+    flex: auto;
+    z-index: 999;
+    display: none;
+    background-color: var(--dark-color-1);
+    backdrop-filter: var(--blur-5);
+    position: absolute;
+    top: 32vh;
+    margin: 0;
+    padding: 1rem;
+    width: 100%;
+    left: 0;
+    box-shadow: var(--bx-sh-primary-1);
+  }
+  .filter-btn {
+    display: block;
+  }
+}
+.show {
+  display: flex !important;
```

Comment:
why is !important used here?

Reply:
Else it can't override the default property